### PR TITLE
fh-mbaas moved to FeedHenry org

### PR DESCRIPTION
**Pull request by psturc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    "sinon": "1.17.0",
+    "supertest": "1.1.0",
+    "turbo-test-runner": "http://npm.skunkhenry.com/turbo-test-runner/-/turbo-test-runner-0.6.3.tgz",
+    "deep-equal": "^1.0.1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:feedhenry/fh-mbaas.git"
+  }
+}
```
Comment (PhilipGough) [inline package.json]: @pavel this looks fine

**Comment by PhilipGough** [comment]
@psturc As discussed on irc we need to create a pr against the fheng repo pointing to this and removing all the code. They will be merged back together then after @grdryn does his thing :)

Also @grdryn  since this is fh-mbaas I'm assuming we need updates tofhcap and possibly some of the jobs on Jenkins also

**Comment by PhilipGough** [comment]
@psturc Have we removed the git history here?

**Comment by psturc** [comment]
@PhilipGough git history should be gone

**Comment by grdryn** [comment]
Retest this please.

**Pull request merged**